### PR TITLE
Update README.md with changes from v7.0.0-beta release.

### PR DESCRIPTION
**Pull request by jhendrixMSFT**

### CODE REVIEW

**Comment by jhendrixMSFT** [comment]
@mcardosos Here's the README update we were discussing.

File: README.md [hunk 1]
```diff
@@ -16,34 +16,48 @@ the packages in your project and update them when a stable tag is out.
 
 [About ARM](/arm/README.md)
 
+- [account] (/arm/datalake-analytics/account)
+            (/arm/datalake-store/account)
```
Comment (mcardosos) [inline README.md]: This line looks weird when rendering the md

File: README.md [hunk 2]
```diff
@@ -16,34 +16,48 @@ the packages in your project and update them when a stable tag is out.
...
+- [analysisservices](/arm/analysisservices)
 - [authorization](/arm/authorization)
 - [batch](/arm/batch)
 - [cdn](/arm/cdn)
 - [cognitiveservices](/arm/cognitiveservices)
+- [commerce](/arm/commerce)
+- [commitmentplans](/arm/machinelearning/commitmentplans)
```
Comment (mcardosos) [inline README.md]: For this packages that are inside other packages, what do you think about including the service they belong to?

File: README.md (hunk 1 above)
Comment (jhendrixMSFT) [inline README.md]: Oh yes I somehow forgot this is markdown, will fix.

File: README.md (hunk 2 above)
Comment (jhendrixMSFT) [inline README.md]: Agreed, fixed.

**Pull request merged**